content/akeneo-event-platform/getting-started.md

```diff
@@ -1,4 +1,161 @@
 # Getting started
 
-::: panel-link <!-- TODO TEXT PANEL LINK -->[Next](/akeneo-event-platform/api-reference.html)
-:::
\ No newline at end of file
+This guide will walk you through the essential steps to set up your environment, create subscriptions, and trigger events.
+
+### Prerequisites
+
+Before proceeding, ensure that you have an active App (it can be a custom App) or an active connection to an Akeneo PIM.
+
+To learn how to create a connection, see the [Authentication Guide](https://api.akeneo.com/documentation/authentication.html#client-idsecret-generation). If you're setting up a custom app, follow the steps in [this tutorial](https://api.akeneo.com/tutorials/how-to-get-your-app-token.html#) to obtain an App token.
+
+---
+
+### Basic Use Case: Setting Up Event Subscriptions
+
+This guide provides a step-by-step walkthrough to help you set up event subscriptions using the Akeneo Event Platform. By the end of this guide, you will have created a connection, authenticated your app, subscribed to events, and triggered an event from your PIM.
+
+### 1. Retrieve your credentials from your targeted PIM
+
+<aside>
+🛠 For every call to the API, you will need `X-PIM-TOKEN` & `X-PIM-CLIENT-ID` .
+
+In this example, we will create a new `connection` in the PIM and use it to generate an `API token`.
+
+</aside>
+
+1. **Create a connection in Akeneo PIM:**
+   - Navigate to **Connect** > **Connection settings** > **Create**.
+   - Fill out the form to create the connection.
+   - Note the generated `Client ID`, `Secret`, `Username`, and `Password`.
+2. **Generate an API Token:**
+   - Encode the `client ID` and `secret` in base64 format, separated by a colon (`:`).
+
+    ```bash
+    client_id:secret
+    4gm4rnoizp8gskgkk080ssoo80040g44ksowwgw844k44sc00s:5dyvo1z6y34so4ogkgksw88ookoows00cgoc488kcs8wk4c40s
+    ```
+
+    ```makefile
+    base64 encoded
```

Comment:
can we provide/document the bash command which get the correct base64 encoded string because I know there is some issues sometimes with encoding with or without padding. In our case if I remember is "with padding which generate some "=" at the end of the string)